ninfod/ninfod.c

```diff
@@ -151,7 +151,8 @@ __inline__ static const char * log_level(int priority) {
 	}
 }
 
-void DEBUG(int pri, char *fmt, ...)
+__iputils_attribute_format(__printf__, 2, 3)
```

Comment:
Sorry to tell you now. I'd use `iputils_attribute_format`. Double underscore at the beginning is reserved for libc internals. Using `__iputils_*` prefix is obviously safe, but also `iputils_*` is safe.